src/colors/button.ts

```diff
@@ -19,6 +19,10 @@ export const button = {
     disabled: {
       DEFAULT: colors.grey['50'],
       dark: background.subtle.dark
+    },
+    selected: {
+      DEFAILT: colors.grey['100'],
```

Comment:
**typo**: there is a typo here and in all the other `selected` objects.  
```suggestion
      DEFAULT: colors.grey['100'],
```

Reply:
All looks good, thanks